### PR TITLE
Update README (WM-1)

### PR DESCRIPTION
**Pull request by akuntaniguchi**
- Add instructions for setting up the application, including environment variables and dependency versions

### CODE REVIEW

**Comment by akuntaniguchi** [comment]
@Kojack8

File: README.md [hunk 1]
```diff
@@ -1,4 +1,26 @@
...
+
+React-Typescript application that changes the background based on the temperature of requested location.
+
+## Setup
+
+### Prerequisites
+
+This project requires the the following tools to be installed before it can be sucessfully built.
+
+| Dependancy | Version    |
```
Comment (Kojack8) [inline README.md]: Dependency*

File: README.md [hunk 2]
```diff
@@ -1,4 +1,26 @@
-# Getting Started with Create React App
+# Weather Map 
+
+React-Typescript application that changes the background based on the temperature of requested location.
+
+## Setup
+
+### Prerequisites
+
+This project requires the the following tools to be installed before it can be sucessfully built.
```
Comment (Kojack8) [inline README.md]: successfully*

File: README.md (hunk 1 above)
Comment (akuntaniguchi) [reply README.md]: Good catch! Fixed spelling

File: README.md (hunk 2 above)
Comment (akuntaniguchi) [reply README.md]: Whoops lol. Fixed!

**Comment by Kojack8** [comment]
+1

**Review by Kojack8 — APPROVED** [review]
+1

**Pull request merged**